supply/lib/supply/setup.rb

```diff
@@ -55,23 +55,22 @@ def download_images(listing, image_format = "png", remove_alpha = false)
 
           image_counter = 1 # Used to prefix the downloaded files, so order is preserved.
           urls.each do |url|
-            url = "#{url}=w6000-h6000" # using huge dimentions intentionally so Google returns the images in their original size they were uploaded
+            url = url.gsub("-rw", "") # Remove "-rw" (if present) so the url will become either 'jpg' or 'png' (https://github.com/fastlane/fastlane/pull/14322#issuecomment-473017130)
+            url = "#{url}=s0" # '=s0' param ensures full image size is returned (https://github.com/fastlane/fastlane/pull/14322#issuecomment-473012462)
```

Comment:
Are we sure we can just add `=` here? If the URL already has a `=` in it, we have to add `-s0` instead :/

Reply:
Good point! This should fix it - https://github.com/fastlane/fastlane/pull/14322/commits/39fca173c86ba56b155b1acaa5fe7fc775771f41